model/metrics/jvm-metrics-experimental.yaml

```diff
@@ -42,9 +42,9 @@ groups:
       - ref: jvm.buffer.pool.name
         requirement_level: recommended
 
-  - id: metric.jvm.buffer.memory.usage
+  - id: metric.jvm.buffer.memory.used
     type: metric
-    metric_name: jvm.buffer.memory.usage
```

Comment:
please keep old metric around (as deprecated).

e.g. this one can be used as an example

https://github.com/open-telemetry/semantic-conventions/blob/21b8f4620d2029fe8abb98a4a06655da3232d11d/model/metrics/deprecated/database.yaml#L2

Reply:
Updated it.